File: app/src/main/java/com/example/tetrisapp/util/MediaPlayerUtil.java
package com.example.tetrisapp.util;

import android.content.Context;
import android.media.MediaPlayer;

public class MediaPlayerUtil {
    private final Context context;

    public MediaPlayerUtil(Context context) {
        this.context = context;
    }

    public void playSound(int resId) {
        playSound(resId, 1f);
    }

    public void playSound(int resId, float volume) {
        final MediaPlayer mediaPlayer = MediaPlayer.create(context, resId);
        mediaPlayer.setOnCompletionListener(MediaPlayer::release);
        mediaPlayer.setVolume(volume, volume);
        mediaPlayer.start();
    }
}

File: app/src/main/java/com/example/tetrisapp/util/MediaPlayerUtil.kt
package com.example.tetrisapp.util

import android.content.Context
import android.media.MediaPlayer

class MediaPlayerUtil(private val context: Context) {
    fun playSound(resId: Int, volume: Float = 1f) {
        val mediaPlayer = MediaPlayer.create(context, resId)
        mediaPlayer.setOnCompletionListener { player -> player.release() }
        mediaPlayer.setVolume(volume, volume)
        mediaPlayer.start()
    }
}